infrastructure-aws-development.yml

```diff
@@ -0,0 +1,13 @@
+instance_groups:
+- name: master
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
+- name: minion
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
```

Comment:
I think the right test would be to redeploy a version of the k8s release that's 1.5 and then deploy the 1.6 release  to see if it was an issue with the upgrade.